integration_test/integration_tests.py

```diff
@@ -22,14 +22,16 @@ def validate(json_to_validate, schema):
     validator.validate(json_to_validate, schema)
 
 
-def send_sms_notification_test_response(python_client):
+def send_sms_notification_test_response(python_client, sender_id=None):
```

Comment:
Add a new test for sender_id being populated.
All of the previous tests shouldn't need updating as the sender_id is optional?

Reply:
This bit here is being used as a helper function for the tests. The test for sending an SMS without a sender id has stayed the same, and lines 157 and 167 now use this function to test sending an SMS with the sender id populated.